content/pages/mua-summary.md

```diff
@@ -0,0 +1,68 @@
+Title: MUA Summary
+Category: pages
+Status: hidden
+Slug: mua-summary
```

Comment:
The title should slugify to this so you don't actually need to specify this. Also, pages isn't a category so you can get rid of that too :)